### PR TITLE
ref(tsdb): Support multiple environments in tsdb.get_range (snuba backend)

### PR DESCRIPTION
**Pull request by macqueen**
needed for organization issues endpoint (which will support selecting multiple environments)

### CODE REVIEW

**Review by wedamija — APPROVED** [review]

File: src/sentry/tsdb/dummy.py [hunk 1]
```diff
@@ -30,7 +30,12 @@ def delete(self, models, keys, start=None, end=None, timestamp=None, environment
             [None])
         self.validate_arguments(models, environment_ids)
 
-    def get_range(self, model, keys, start, end, rollup=None, environment_id=None):
+    def get_range(self, model, keys, start, end, rollup=None, environment_ids=None):
+        # only snuba backend supports multiple envs
```
[the remark this answers is not in the document]
Comment (macqueen) [reply src/sentry/tsdb/dummy.py]: good call

File: src/sentry/tsdb/inmemory.py [hunk 2]
```diff
@@ -73,7 +73,12 @@ def delete(self, models, keys, start=None, end=None, timestamp=None, environment
                                 0,
                             )
 
-    def get_range(self, model, keys, start, end, rollup=None, environment_id=None):
+    def get_range(self, model, keys, start, end, rollup=None, environment_ids=None):
+        # only snuba backend supports multiple envs
```
[the remark this answers is not in the document]
Comment (macqueen) [reply src/sentry/tsdb/inmemory.py]: i think to be able to load the events stream / discover, you'd need to have snuba runnign locally, so i could see it being a requirement. let me see how annoying it'd be to support for now though

[the remark this answers is not in the document]
Comment (macqueen) [reply src/sentry/tsdb/inmemory.py]: ok didn't wind up being too hard, so added support

[the remark this answers is not in the document]
Comment (wedamija) [reply src/sentry/tsdb/inmemory.py]: Nice! Everything still lgtm

**Pull request merged**